save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt

```diff
@@ -13,24 +15,24 @@ class TestExecutionService(private val testResultRepository: TestExecutionReposi
     private val log = LoggerFactory.getLogger(TestExecutionService::class.java)
 
     /**
-     * @param testExecutions list of test executions
+     * @param testExecutionsDto
      * @return list of lost tests
      */
-    fun saveTestResult(testExecutions: List<TestExecution>): List<TestExecution> {
-        val lostTests: MutableList<TestExecution> = mutableListOf()
-        testExecutions.forEach { testExec ->
-            val foundTestExec = testResultRepository.findById(testExec.id)
+    fun saveTestResult(testExecutionsDto: List<TestExecutionDto>): List<TestExecutionDto> {
```

Comment:
```suggestion
    fun saveTestResult(testExecutionsDtos: List<TestExecutionDto>): List<TestExecutionDto> {
```
Naming collections with plural names makes the code much more readable, also you won't need to abbreviate identifier inside lambda (`testExecDto`)